src/reference/command_output_tags.adoc

```diff
@@ -2,6 +2,50 @@
 
 Commands can print tags to stdout to communicate with Cog. To use one, print it
 to stdout at the beginning of a newline before returning your actual response.
+If you're not returning JSON output (by using the JSON tag), you may
+intersperse tags in along with lines of output if necessary.
+
+=== Examples
+
+Output from a command that includes a debug statement, template selection, and
+JSON response. The debug statement will be printed in the logs of the Relay it
+ran on, while the JSON output will be rendered in template specified and sent
+to chat.
+
+----
+COGCMD_DEBUG: Requst took 53ms to complete
+COG_TEMPLATE: instances
+JSON
+[
+  {
+    "instance_id": "i-1984f3",
+    "instance_type: "t2.micro"
+  },
+  {
+    "instance_id": "i-2a7b11",
+    "instance_type: "t2.small"
+  }
+]
+----
+
+Output from a command in freeform text which also includes two info statements
+and also uses a template. The info statement will be printend in the logs of
+the Relay it ran on, while the text output will be rendered in the template
+specified and sent to chat. Even though tags were interspersed in the text
+output, each line that does not start with a tag, will be combined to form a
+single string, newlines included. That string will then be turned into a JSON
```

Comment:
I believe that instead of a single line with embedded `\n` characters, the output is converted into an array of strings, (one per line), and that array is then set as the body.